docs/backup-restore.md

```diff
@@ -67,6 +68,10 @@ Use the `.backup` command with the `.timeout` command as specified in [Special c
 
 For an example, see [BASH: SQLite3 .backup command](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command).
 </div>
+<div id="YugabyteDB_Managed1" class="tabcontent" markdown="1">
+
+Clusters are backed up automatically based on the backup policy, and these backups are retained for a specific duration. You can also perform on-demand backups. See [YugabyteDB Managed: Back up and restore clusters](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/).
```

Comment:
```suggestion
Clusters are backed up automatically based on the backup policy, and these backups are retained for a specific duration. You can also perform on-demand backups, as described in [YugabyteDB Managed: Back up and restore clusters](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/).
```

Reply:
@josh-wong Thanks for the suggestion! I basically agree on it.
My slight concern is that the YugabyteDB doc is also helpful to understand scheduled backups not only on-demand backups. With the change you suggested, it seems the doc is only related to on-demand ones?

Reply:
Oh, right😅 How about we use the following instead?

```suggestion
Clusters are backed up automatically based on the backup policy, and these backups are retained for a specific duration. You can also perform on-demand backups. For details on performing backups, see [YugabyteDB Managed: Back up and restore clusters](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/).
```

Reply:
@josh-wong 💯

Reply:
@josh-wong Can I apply this to other sentences as well? Or you've been already working on it?

Reply:
Sorry for the slow reply. I've updated my suggestions!

Reply:
@josh-wong Thanks a lot!